docs/src/pinn/system.md

```diff
@@ -80,26 +82,156 @@ ts,xs = [infimum(d.domain):0.01:supremum(d.domain) for d in domains]
 initθ = discretization.init_params
 acum =  [0;accumulate(+, length.(initθ))]
 sep = [acum[i]+1 : acum[i+1] for i in 1:length(acum)-1]
-minimizers = [res.minimizer[s] for s in sep]
+minimizers_ = [res.minimizer[s] for s in sep]
 
 analytic_sol_func(t,x) = [exp(-t)*sin(pi*x), exp(-t)*cos(pi*x), (1+pi^2)*exp(-t)]
 u_real  = [[analytic_sol_func(t,x)[i] for t in ts for x in xs] for i in 1:3]
-u_predict  = [[phi[i]([t,x],minimizers[i])[1] for t in ts  for x in xs] for i in 1:3]
+u_predict  = [[phi[i]([t,x],minimizers_[i])[1] for t in ts  for x in xs] for i in 1:3]
 diff_u = [abs.(u_real[i] .- u_predict[i] ) for i in 1:3]
-
 for i in 1:3
-    p1 = plot(ts, xs, u_real[i], st=:surface,title = "u$i, analytic");
-    p2 = plot(ts, xs, u_predict[i], st=:surface,title = "predict");
+    p1 = plot(ts, xs, u_real[i],linetype=:contourf,title = "u$i, analytic");
+    p2 = plot(ts, xs, u_predict[i],linetype=:contourf,title = "predict");
+    p3 = plot(ts, xs, diff_u[i],linetype=:contourf,title = "error");
+    plot(p1,p2,p3)
+    savefig("sol_uq$i")
+end
+```
+
+![sol_uq1](https://user-images.githubusercontent.com/12683885/122979254-03634e80-d3a0-11eb-985b-d3bae2dddfde.png)
+
+![sol_uq2](https://user-images.githubusercontent.com/12683885/122979278-09592f80-d3a0-11eb-8fee-de3652f138d8.png)
+
+![sol_uq3](https://user-images.githubusercontent.com/12683885/122979288-0e1de380-d3a0-11eb-9005-bfb501959b83.png)
+
+
+## Derivative neural network approximation
```

Comment:
what does that mean?

Reply:
we approximate the derivative of the neural network with another neural network `Dt(u1(t,x)) ~ Dtu1(t,x)` thus we avoid taking the second derivative `Dt(Dtu1(t,x)) `.
Since `u3` is only in the first and second equations, that its accuracy during training is determined by the accuracy of the second numerical derivative `Dtt(u1(t,x)) ~ Dxx(u1(t,x)) + u3(t,x)*sin(pi*x)`.
So this is such the trick to not take the second numeric derivative that allows an increase in the marginal precision for optimization.

Reply:
ahh this needs more of an explanation.

Reply:
https://github.com/SciML/NeuralPDE.jl/blob/Systems_of_PDE_fix/docs/src/pinn/system.md#comparison-of-the-second-numerical-derivative-and-numerical--neural-network-derivative

Reply:
But there's no explanation. There's a title, and then there's no text explaining what the section is actually about. There needs to be some introductory text as to what you're doing there and why.

Reply:
oh, I have thought you need more explanation :) 
ok, I will add a text explanation to the doc.